voice/bxml/verbs/speakSentence.md

```diff
@@ -42,6 +42,7 @@ The table below shows a mapping of our supported voices.  It also maps our voice
 | esperanza | es         | female     | Conchita   |
 | violeta   | es         | female     | Lucia      |
 | jorge     | es         | male       | Enrique    |
+| rosa      | es_MX      | female     | Mia        |
```

Comment:
```suggestion
| rosa      | es_mx      | female     | Mia        |
```
We should be consistent. I'm also fine with capitalizing all of the country codes, since I think it's more correct.